libs/core/iterator_support/tests/include/hpx/iterator_support/tests/iterator_tests.hpp

```diff
@@ -344,7 +344,8 @@ namespace tests {
     template <typename Iterator, typename TrueVals>
     void random_access_readable_iterator_test(Iterator i, int N, TrueVals vals)
     {
-        bidirectional_readable_iterator_test(i, vals[0], vals[1]);
+        bidirectional_readable_iterator_test(
+            i, vals[0], N > 1 ? vals[1] : vals[0]);
```

Comment:
Huh, why? https://github.com/STEllAR-GROUP/hpx/pull/5703 asserts that `N` is at least 2 because the iterator `i` is advanced and compared to `v2` in the tests.

Reply:
Uhh, merge snafu, I selected the wrong change here. Will fix.